File: loop/1115.py
#1115 QUADRANTE
while True:
    valores = input().split()
    x, y = map(int, valores)
    if x == 0 or y == 0:
        break
    elif x > 0 and y > 0:
        print("primeiro")
    elif x < 0  and y > 0:
        print("segundo")
    elif x < 0 and y < 0:
        print("terceiro")
    elif x > 0 and y < 0:
        print("quarto")

File: javascript/1115.js
/*Quadrante*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split("\n");

let linhas = lines.shift();
let valores = linhas.split(" ");
let x = parseInt(valores.shift());
let y = parseInt(valores.shift());

while(x !== 0 && y !== 0){
  if(x > 0 && y > 0){
    console.log("primeiro");
  }else if(x < 0 && y > 0){
    console.log("segundo");
  }else if(x < 0 && y < 0){
    console.log("terceiro");
  }else if(x > 0 && y < 0){
    console.log("quarto");
  }
linhas = lines.shift();
valores = linhas.split(" ");
x = parseInt(valores.shift());
y = parseInt(valores.shift());
}